practice_app/pages/index.js

```diff
@@ -187,6 +192,30 @@ export default function Home({ context }) {
         }
     }
 
+    // This function returns the translated version of the input, to the language spoken in user's location
+    // current functionality : source_lang=english, target_lang=ip_dependent
+    const getTranslation = async () => {
+        // Learn client's ip adress using a third party service
+        const ipinfo = await axios.get('https://json.geoiplookup.io/')
```

Comment:
Please use camelCase for variable names. `ipInfo`

Reply:
Typo corrected.